### DISCUSSION TITLE
Link not Working in Documentation

### DISCUSSION THREAD

**Discussion by sushantbasak** [loopbackio/loopback-next#8273] [Q&A]
![image](https://user-images.githubusercontent.com/56016930/151773616-5f5d3e1b-b9aa-46e5-9b70-50db55fdb4f2.png)

**In this Page, the link that is present in DataSource Documentation is not working.**

**[Page Link](https://github.com/loopbackio/loopback-next/blob/master/docs/site/Creating-datasource-runtime.md
)**
![Image](https://user-images.githubusercontent.com/56016930/151773912-fcc0e7f7-428d-4dd9-a081-3a102761865d.png)

**Comment by dhmlau** [comment]
@sushantbasak, thanks for reporting this. 
The url should begin with apidocs.loopback.io instead of apicdocs.strongloop.com. http://apidocs.loopback.io/loopback-datasource-juggler/#datasource
Would you like to submit a PR?

**Reply by sushantbasak** [reply]
I am new at Open Source.It will be great opportunity for me to address the Issue by submitting a PR. Can you let me know, do I need to make an Issue first and then PR or directly PR?

**Reply by mrmodise** [reply]
@sushantbasak nice of you to be willing to do a PR. I think you can do a PR directly and link it by commenting with the #8273 on your PR

**Reply by sushantbasak** [reply]
Got it.

**Reply by sushantbasak** [reply]
I have created a PR #8286 to fix this Issue.

**Comment by dhmlau** [comment]
There are actually 2 more occurrances: https://github.com/loopbackio/loopback-next/search?q=%22apidocs.strongloop.com%22.
